dashboard/app/controllers/policy_compliance_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PolicyComplianceController < ApplicationController
+  # GET /policy_compliance/child_account_consent
+  # URL where parents or guardians give consent for their child to have an
+  # account on Code.org
+  # Our child account policy requires that parents give us permission to create
+  # a Code.org account for their child if the child meets the following
+  # criteria:
+  #  * The child is under 13 years old.
+  #  * The child is creating an account using an email address.
+  #  * The child lives in Colorado.
+  def child_account_consent
+    @permission_granted = false
+    token = request.params[:token]
+    return render status: :bad_request if token.blank?
+    #Get parent permission request
+    permission_request = ParentalPermissionRequest.find_by(uuid: token)
```

Comment:
`params.require(:token)` will raise `ActionController::ParameterMissing` if `token` is blank and return `bad_request` HTTP status.
```suggestion
    #Get parent permission request
    permission_request = ParentalPermissionRequest.find_by(uuid: params.require(:token))
```

Reply:
Oh I didn't know about this. Thanks!